src/main/scala/sangria/slowlog/MetricRenderer.scala

```diff
@@ -25,7 +25,7 @@ object MetricRenderer {
 class DefaultMetricRenderer(val unit: TimeUnit) extends MetricRenderer {
   def renderVariables[In: InputUnmarshaller](variables: In, names: Vector[String]) = {
     val iu = implicitly[InputUnmarshaller[In]]
-    val renderedVars = names.flatMap(name =>
+    val renderedVars = names.iterator.flatMap(name =>
```

Comment:
found during profiling, no need to allocate a collection for reducing it with `mkString`